leandra-project/src/components/PullSingleRepo.js

```diff
@@ -6,7 +6,7 @@ const PullSingleRepo = (props) => (
     <ul>
 
         {props.reposToList.map(repo => <li key={repo.id}><a href={repo.payload.pull_request.html_url} target="_blank" rel="noopener noreferrer">{repo.payload.pull_request.title}</a>
-        <PullRequestStatus status={repo.payload.pull_request.url} />
+        <PullRequestStatus status={repo.payload.action} merged={repo.payload.pull_request.merged} />
```

Comment:
I've noticed there are some pull-request payload that can be found in `repo.payload.pull_request`. 

So I placed a `merge` prop here.